File: src/targets/rust/reqwest/fixtures/jsonObj_multiline.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "http://mockbin.com/har";

    let payload = "{\n  \"foo\": \"bar\"\n}";

    let mut headers = reqwest::header::HeaderMap::new();
    headers.insert("content-type", "application/json".parse().unwrap());

    let client = reqwest::Client::new();
    let response = client.post(url)
        .body(payload)
        .headers(headers)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    dbg!(results);
}

File: src/targets/dart/http/fixtures/jsonObj-multiline.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'content-type': 'application/json',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
    body: "{\n  \"foo\": \"bar\"\n}",
  );

  print(response.statusCode);
  print(response.body);

}